tracing-subscriber/src/fmt/mod.rs

```diff
@@ -873,6 +873,15 @@ impl<N, E, F, W> SubscriberBuilder<N, E, F, W> {
             inner: self.inner.with_writer(make_writer),
         }
     }
+
+    /// Prints text using the standard library's macro `print!`. This writer allows stdout logs to
+    /// be correctly captured by commands such as `cargo test`.
+    pub fn with_test_writer(self) -> SubscriberBuilder<N, E, F, TestWriter> {
+        SubscriberBuilder {
+            filter: self.filter,
+            inner: self.inner.with_writer(TestWriter),
+        }
+    }
```

Comment:
I'd like to see this method also be implemented on [fmt::Layer](https://docs.rs/tracing-subscriber/0.2.11/tracing_subscriber/fmt/struct.Layer.html#implementations).